File: pasabaybcd/split_json.js
const fs = require('fs');
const path = require('path');

const seedFile = path.join(__dirname, 'mongodb_seed.json');
try {
  const data = fs.readFileSync(seedFile, 'utf8');
  const json = JSON.parse(data);
  
  for (const [key, value] of Object.entries(json)) {
    if (Array.isArray(value)) {
      const outFile = path.join(__dirname, `${key}.json`);
      fs.writeFileSync(outFile, JSON.stringify(value, null, 2));
      console.log(`Created ${key}.json with ${value.length} records.`);
    }
  }
} catch (e) {
  console.error("Error:", e);
}

File: pasabaybcd/split_json.py
import json
import os

def split_json():
    seed_file = 'mongodb_seed.json'
    if not os.path.exists(seed_file):
        print(f"Error: {seed_file} not found")
        return

    with open(seed_file, 'r') as f:
        data = json.load(f)

    for collection_name, collection_data in data.items():
        if isinstance(collection_data, list):
            out_file = f"{collection_name}.json"
            with open(out_file, 'w') as f:
                json.dump(collection_data, f, indent=2)
            print(f"Created {out_file} with {len(collection_data)} records.")

if __name__ == '__main__':
    split_json()
